packages/application/style/menus.css

```diff
@@ -31,7 +31,12 @@
 }
 
 .lm-MenuBar-item {
-  padding: 0 8px;
+  /* TODO: remove ignore directive when fixed: https://github.com/csstree/csstree/issues/164 */
+  /* stylelint-disable-next-line csstree/validator */
+  padding: 0 min(calc(4vw - 11.2px), 8px); /* The use of min(), calc() and vw units
+    has the effect that as the screen drops below 480px, horizontal padding will
```

Comment:
Note: 480px was chosen through trial and error. At around 420px with 8px padding between menu items, the menu bar starts to be too wide for the screen width. 480px gives a little wiggle room.

This gave two equations to solve (units in pixels):

```
(480px)a + b = 8
(320px)a + b = 1.5
```

Because at 320px, the padding needs to be set to 1.5px for the items to fit. Solving the equations gives:

```
a ~= 0.04 (equivalent to 4vw)
b ~= -11.2px
```